Project.toml

```
@@ -28,17 +28,17 @@ cohomCalg_jll = "5558cf25-a90e-53b0-b813-cadaa3ae7ade"
 lib4ti2_jll = "1493ae25-0f90-5c0e-a06c-8c5077d6d66f"
 
 [compat]
-AbstractAlgebra = "0.46.2"
+AbstractAlgebra = "0.47.0"
 AlgebraicSolving = "0.9.0"
 Compat = "4.13.0"
 Distributed = "1.6"
 GAP = "0.15.0"
-Hecke = "0.37.2"
+Hecke = "0.38.0"
```

```suggestion
Hecke = "0.38.1"
```
apply once https://github.com/thofma/Hecke.jl/commit/e90a02fabe1ea3897df4a370bfbebb870196fa40 is released and available (ping @thofma )